src/main/java/uk/gov/hmcts/darts/cache/token/service/value/impl/RefeshableTokenCacheValue.java

```diff
@@ -13,6 +13,10 @@
 
 import java.util.Optional;
 
+/**
+ * A default implementation of the {@link DownstreamTokenisableValue} that uses {@link TokenGeneratable} to generate
```

Comment:
Please can you rename the typo of the class field - `jwtCacheRegistrable`
All these "able"s hurt my head a bit, but it maybe just needs to end in "Registerable"?